emitterc.go

```diff
@@ -361,6 +361,7 @@ func (emitter *yamlEmitter) emitStreamStart(event *yamlEvent) bool {
 // Expect DOCUMENT-START or STREAM-END.
 func (emitter *yamlEmitter) emitDocumentStart(event *yamlEvent, first bool) bool {
 	if event.typ == yaml_DOCUMENT_START_EVENT {
+		isEmpty := emitter.events_head+1 < len(emitter.events) && emitter.events[emitter.events_head+1].typ == yaml_DOCUMENT_END_EVENT
```

Comment:
I woulije to suggest something Luke thus that could be clearer and Kore idiomatic maybe


```suggestion
		var isEmpty bool
		next, found :=  emitter.events[emitter.events_head+1]
		if found && next.typ == yaml_DOCUMENT_END_EVENT {
			isEmpty = true
		}
```

Reply:
I don't think this works (https://stackoverflow.com/a/64415778).

Reply:
Oh you are right